### PR TITLE
Adding simple e2e test to presubmit 

### PR DESCRIPTION
**Pull request by hklai**
Fixes https://github.com/istio/istio/issues/3509

### CODE REVIEW

File: prow/e2e-simpleTests.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright 2017 Istio Authors
```
Comment (mattdelco) [inline prow/e2e-simpleTests.sh]: 2018.

Offhand it looks like prow/config.yaml in the test-infra repo might need to be updated to reflect the rename of e2e-smoke.sh.

Comment (hklai) [reply prow/e2e-simpleTests.sh]: Yup. https://github.com/istio/test-infra/pull/694/

**Comment by yutongz** [comment]
nit: make prow/e2e-simpleTests.sh executable

**Comment by ldemailly** [comment]
ty!

**Comment by ldemailly** [comment]
but it failed?

**Comment by hklai** [comment]
both bookInfo and simple tests passed in this PR. Where do you see the failures?

**Comment by ldemailly** [comment]
it was the link/failure above - but somehow the error changed or I misremembered

https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_istio/3529/e2e-smoke/3031/

**Comment by hklai** [comment]
That happened in the time window between the two PRs I merged.

https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_istio/3557/e2e-simpleTests/36/
is probably a more real error, but it also means the simple test is flaky.

On Fri, Feb 16, 2018 at 12:44 AM, Laurent Demailly <notifications@github.com

**Pull request merged**